addons/purchase/models/product.py

```diff
@@ -40,11 +41,13 @@ def get_import_templates(self):
 
     def action_view_po(self):
         action = self.env.ref('purchase.action_purchase_order_report_all').read()[0]
-        action['domain'] = ['&', ('state', 'in', ['purchase', 'done']), ('product_tmpl_id', 'in', self.ids)]
+        action['domain'] = [
```

Comment:
It seems you did not change the line except removing the '&', am I wrong ? Otherwise keep diff minimal by avoiding rewriting code :) .

Reply:
Indeed, I did not know about keeping changes minimal. I did this as it is & by default...